specs/_features/maxeb_increase/capella.py

```diff
@@ -1681,6 +1734,47 @@ def for_ops(operations: Sequence[Any], fn: Callable[[BeaconState, Any], None]) -
     for_ops(body.deposits, process_deposit)
     for_ops(body.voluntary_exits, process_voluntary_exit)
     for_ops(body.bls_to_execution_changes, process_bls_to_execution_change)  # [New in Capella]
+    for_ops(body.execution_payload.withdraw_request, process_execution_layer_withdraw_request)
+
+
+def process_execution_layer_withdraw_request(
+        state: BeaconState,
+        execution_layer_withdraw_request: ExecutionLayerWithdrawRequest
+    ) -> None:
+    validator_pubkeys = [v.pubkey for v in state.validators]
+    validator_index = ValidatorIndex(validator_pubkeys.index(execution_layer_withdraw_request.validator_pubkey))
+    validator = state.validators[validator_index]
+
+    # Same conditions as in EIP7002 https://github.com/ethereum/consensus-specs/pull/3349/files#diff-7a6e2ba480d22d8bd035bd88ca91358456caf9d7c2d48a74e1e900fe63d5c4f8R223
+    # Verify withdrawal credentials
+    is_execution_address = validator.withdrawal_credentials[:1] == ETH1_ADDRESS_WITHDRAWAL_PREFIX
+    is_correct_source_address = validator.withdrawal_credentials[12:] == execution_layer_withdraw_request.source_address
+    if not (is_execution_address and is_correct_source_address):
+        return
```

Comment:
I think it's better to assert here and declare the block as invalid.

Reply:
It should be a noop rather than assert to skip invalid withdrawal/exist request as EL can't know whether a request is invalid and will keep including it in the next block.

Reply:
The entire solution involves a queue in the EL's world state elements of which are dequeued every built block, block invalidation instead of a no op would lead a system into an indefinite loop.